src/main/menu.ts

```diff
@@ -217,4 +217,35 @@ export function buildMenu(windowManager: WindowManager) {
   mt.push(helpMenu)
 
   Menu.setApplicationMenu(Menu.buildFromTemplate(mt));
+
+  if (!!process.env.JEST_WORKER_ID) {
```

Comment:
What is test code doing inside `menu.ts`? :) 
Also we have btw flag from `common/vars.ts` for checking test env: `isTestEnv`